generic/rcutorture.py

```diff
@@ -39,16 +40,13 @@ def setUp(self):
         """
         self.results = []
         self.log.info("Check if CONFIG_RCU_TORTURE_TEST is enabled\n")
-        ret = os.system(
-            'cat /boot/config* | grep CONFIG_RCU_TORTURE_TEST=m')
-        if ret != 0:
+        ret = linux_modules.check_kernel_config('CONFIG_RCU_TORTURE_TEST')
+        if ret == 0:
```

Comment:
@abdhaleegit  yes ` linux_module.check_kernel_config` return 0 , 1 ,2 but i think in this API we have a global variable as `NOT_SET,MODULE,BUILTIN `respectively `0,1,2 `

oh sorry  we can use` linux_modules.NOT_SET`